lib/ansible/modules/find.py

```diff
@@ -100,6 +100,30 @@
         type: str
         choices: [ atime, ctime, mtime ]
         default: mtime
+    permissions:
```

Comment:
users and groups seem fine, but permissions ...  the implementation leaves out octal and also seems to be very limited to what it supports when using symbolic mode. I would almost prefer to leave it out and let people reduce the output via jinja filters than push the implementation in this state.

Reply:
I'd agree, with the author of #76586, that having an option to easily find executable files would make a lot of tasks easier. You could surely do it with Jinja, but that applies to almost any argument of this module.

Would it be an option for you, if I'd add support for octal definitions using the same parameter?

Reply:
Removed the feature as it has been merged in https://github.com/ansible/ansible/commit/ca08261f08a5071cc5f8c73e61342f5a9581b9cd